### ISSUE TITLE
C93 1.29.x (Nightly) manual pass/regression check

### ISSUE THREAD

**Issue by kjozwiak** [brave/qa-resources#358]
#### Release Date/Target:

* Release Date: **`ASAP once C93 is merged into master`**

#### Summary:

QA needs to run through **`C92`** on Nightly before it's uplifted into the other channels (targeting `1.27.x`)
#### Manual Passes/Documentation:

* https://github.com/brave/brave-browser/issues/17521
